.github/workflows/vib-verify.yaml

```diff
@@ -93,4 +93,48 @@ jobs:
           # Target-Platform used by default
           VIB_ENV_TARGET_PLATFORM: ${{ secrets.VIB_ENV_TARGET_PLATFORM }}
           # Alternative Target-Platform to be used in case of incompatibilities
-          VIB_ENV_ALTERNATIVE_TARGET_PLATFORM: ${{ secrets.VIB_ENV_ALTERNATIVE_TARGET_PLATFORM }}
\ No newline at end of file
+          VIB_ENV_ALTERNATIVE_TARGET_PLATFORM: ${{ secrets.VIB_ENV_ALTERNATIVE_TARGET_PLATFORM }}
+  ci-pr-review:
+    runs-on: ubuntu-latest
+    needs: vib-verify
+    if: |
+      always() &&
```

Comment:
nit-pick: Is this equivalent to `if true &&`? Can be dropped?

Reply:
Yes, it is equivalent. In this case, we can't drop it as we want our `ci-pr-review` job to run after the `vib-verify` one runs (defined by `needs: vib-verify`) regardless of successful execution of `vib-verify`. Without it, this last job would be skipped if the previous one failed.

More info at: [GitHub Actions doc](https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#defining-prerequisite-jobs)

Reply:
Ah! I get it now. So it means "run regardless of previous step failures"
Thanks!